src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotation.java

```diff
@@ -3,17 +3,12 @@
 import java.util.List;
 
 /**
- * Superclass of all variant annotations.
+ * Interface of all variant annotations. See also InfoFieldAnnotation, GenotypeFieldAnnotation and AbstractInfoFieldAnnotation
  */
-public abstract class VariantAnnotation implements Annotation{
+public interface VariantAnnotation extends Annotation {
 
     /**
      * Return the keys
      */
-    public abstract List<String> getKeyNames();
-
-    @Override
-    public String toString() {
-        return getClass().getSimpleName();
-    }
```

Comment:
Removing this method means that other subclasses, such as `GenotypeAnnotation` will no longer have this override as well. See my overall comments.